### ISSUE TITLE
Support for CCU-Jack devices

### ISSUE THREAD

**Issue by FraatTailscale** [ioBroker/ioBroker.hm-rpc#1003]
**Is your feature request related to a problem? Please describe.**
I cannot see virtual CCU-Jack devices in RPC-Virtual devices.
https://github.com/mdzio/ccu-jack 

**Describe the solution you'd like**
I would like to be able to see CCU-Jack devices in RPC-Virtual devices

**Describe alternatives you've considered**
Don't found a alternative

**Comment by foxriver76** [comment]
as far as I understand this from the README, this is one single API to connect to all CCU devices, instead of e.g. 5 RPC connections for different device types. This would then be better an adapter request for a adapter, because hm-rpc of course is focused on RPC only and the concept also does not fit in this adapter.

**Comment by FraatTailscale** [comment]
as I have understood CCU-Jack is creating Homematic devices like CUxD is doing this, just with another interface category. 


![2023-09-08_10h55_56](https://github.com/ioBroker/ioBroker.hm-rpc/assets/100598566/bbf2abbd-b2c8-448b-969e-d60d5f529803)

**Comment by Apollon77** [comment]
@FraatTailscale Yes but also cuxd is offiéring all details on a special port but reuses the official protocols that hometaic uses too.
But CCU-jack is a bridge to bring in devices buzt in fact control them via REST or MQTT ... so the goal is better to connect to these interfaces directly.

For the case where CCU-Jack "adds virtual devices into CCU" they come from other sources via CCU-jack ... honestly, also connect to these devices directly.

**Comment by 63chris** [comment]
yes, this would be indeed of great help to get the states directly into ioBroker. This add-on is getting more and more popular in the HM world.

Currently it's at least possible to use @ptweety/node-red--contibute-ccu to read/write CCU-Jack states. This, however require usage of user variables.

**Comment by gituser-rk** [comment]
I believe this adapter needs another daemon section "CCU-Jack" like the sections for rfd, CUxD, Homegear to support the RPC Interface of CCU-Jack at xmlrpc://{IP der CCU}:2121/RPC3
Quote from CCU-Jack developer:
"Der CCU-Jack bietet eine kompatible XML-RPC-Schnittstelle (auf der Adresse xmlrpc://{IP der CCU}:2121/RPC3) für alle virtuellen Jack-Geräte. Allein dadurch kann CCU selbst die Geräte überhaupt abfragen und steuern.

Da muss dann tatsächlich ioBroker erweitert werden. Allerdings ist dies nur eine weitere XML-RPC-Schnittstelle, die bereits für z.B. HmIP vorhanden ist."
see https://github.com/mdzio/ccu-jack/issues/220#issuecomment-3627170065
